certified-connectors/TOPdesk/apiDefinition.swagger.json

```diff
@@ -0,0 +1,11510 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "A Connector for TOPdesk, which entails connectors for Supporting Files, and Incident Management. All endpoint for creating, processing and closing incidents are available in the connector, aswell as alle endpoints for creating and handling branches, locations, persons and operators.",
+    "version": "1.0",
+    "title": "TOPdesk API V1",
+    "contact": {
+      "name": "TOPdesk",
+      "url": "http://www.topdesk.com",
+      "email": "info@topdesk.com"
+    }
+  },
+  "basePath": "/tas/api",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata":[
+  {
+  "propertyName": "Website",
+  "propertyValue": "https://www.topdesk.com/"
+  },
+  {
+  "propertyName": "Privacy Policy",
+  "propertyValue": "https://www.topdesk.com/en/privacy-policy/"
+  },
+  {
+  "propertyName": "Categories",
+  "propertyValue": "Service;Helpdesk;ITSM;IT Operations;ESM"
+  }
+  ]
+  "tags": [
+    {
+      "name": "Incident"
+    },
+    {
+      "name": "General"
+    },
+    {
+      "name": "Time Registration"
+    },
+    {
+      "name": "Progress trail / Actions / Requests"
+    },
+    {
+      "name": "Escalation / Deescalation"
+    },
+    {
+      "name": "Archive / Unarchive"
+    },
+    {
+      "name": "Attachments"
+    },
+    {
+      "name": "Branches"
+    },
+    {
+      "name": "Departments"
+    },
+    {
+      "name": "Locations"
+    },
+    {
+      "name": "Operator groups"
+    },
+    {
+      "name": "Operators"
+    },
+    {
+      "name": "Persons"
+    }
+  ],
+  "paths": {
+    "/incidents": {
+      "x-reporting": {
+        "name": "Incident",
+        "pluralName": "Incidents",
+        "singleEntityUrl": "/incidents/id/{id}",
+        "defaultQueryParameters": "dateFormat=iso8601"
+      },
+      "x-api": {
+        "pageStartAlias": "start",
+        "pageSizeAlias": "page_size",
+        "recommendedPageSize": 5000,
+        "defaultSortField": "id"
+      },
+      "get": {
+        "tags": [
+          "Incident"
+        ],
+        "summary": "Returns a list of incidents.",
+        "description": "| Permission | Remarks |\n| ------- | --------|\n| Operator   |  With read permission on 1st/2nd line incident; Category/Branch/Operator filters apply  |\n| Person   |  Accessible; Person visibility settings apply |\n",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "pageStart",
+            "description": "The offset to start at. The default is 0.\n",
```

Comment:
@CJusjong Why the property description values are ended with a new line? It is not required, please remove it.